specification/appconfiguration/AppConfiguration/routes.tsp

```diff
@@ -9,6 +9,13 @@ using Azure.Core.Traits;
 
 namespace AzureAppConfiguration;
 
+alias Body<T, TName extends valueof string = "body", TDoc extends valueof string = "The body parameter."> = {
+  @doc(TDoc)
+  @friendlyName(TName)
```

Comment:
Do not use friendly name like that

Reply:
If the issue we're trying to solve is only for client libraries why arent we just using clientName?